src/pages/fileCentricCart/cartView.js

```diff
@@ -0,0 +1,524 @@
+    const todaysDate = `${yyyy}-${mm}-${dd}`;
+
+    let hours = date.getHours();
+    let minutes = date.getMinutes();
+    let seconds = date.getSeconds();
+
+    if (hours < 10) { hours = `0${hours}`; }
+
+    if (minutes < 10) { minutes = `0${minutes}`; }
+
+    if (seconds < 10) { seconds = `0${seconds}`; }
+
+    return `${'ICDC File Manifest'} ${todaysDate} ${hours}-${minutes}-${seconds}${'.csv'}`;
+  }
+
+  function convertToCSV(jsonse) {
+    const objArray = jsonse;
+    const array = typeof objArray !== 'object' ? JSON.parse(objArray) : objArray;
+    let str = '';
+    array.map((entry, index) => {
+      const keysToInclude = ['case_id', 'file_name', 'uuid', 'md5sum'];
+      let line = '';
+
+      keysToInclude.map((keyName) => {
+        if (line !== '') line += ',';
+        line += entry[keyName];
+        return line;
+      });
+
+      if (index === 0) {
+        str = ['Case ID', 'File Name', 'File ID', 'Md5sum', 'User Comments'].join(',');
+        str += `\r\n${line},${document.getElementById('multiline-user-coments').value}\r\n`;
+      } else {
+        str += `${line}\r\n`;
+      }
+      return str;
+    });
+    return str;
+  }
+
+  function downloadJson() {
+    const jsonse = JSON.stringify(data);
+    const csv = convertToCSV(jsonse);
+    const exportData = new Blob([csv], { type: 'text/csv' });
+    const JsonURL = window.URL.createObjectURL(exportData);
+    let tempLink = '';
+    tempLink = document.createElement('a');
+    tempLink.setAttribute('href', JsonURL);
+    tempLink.setAttribute('download', fileName());
+    document.body.appendChild(tempLink);
+    tempLink.click();
+    document.body.removeChild(tempLink);
+  }
+
+  function onRowsSelect(curr, allRowsSelected) {
+    globalData = [];
+    selectedFileIDs = [];
+    allRowsSelected.forEach((row) => {
+      const file = data[row.dataIndex];
+      selectedFileIDs.push(file.uuid);
+      globalData.push({
+        caseId: file.case_id,
+        fileName: file.file_name,
+        uuid: file.uuid,
+        md5Sum: file.md5sum,
+      });
+    });
+    // filter out the duplicate file ids.
+    selectedFileIDs = [...new Set(selectedFileIDs)];
+    if (allRowsSelected.length === 0) {
+      deleteButton.current.disabled = true;
+      deleteButton.current.style.color = '#FFFFFF';
+      deleteButton.current.style.backgroundColor = '#C53B27';
+      deleteButton.current.style.opacity = '0.3';
+      deleteButton.current.style.border = '3px solid grey';
+      deleteButton.current.style.fontWeight = '600';
+      deleteButton.current.style.cursor = 'auto';
+    } else {
+      deleteButton.current.disabled = false;
+      deleteButton.current.style.color = '#FFFFFF';
+      deleteButton.current.style.backgroundColor = '#C53B27';
+      deleteButton.current.style.cursor = 'pointer';
+      deleteButton.current.style.opacity = 'unset';
+      deleteButton.current.style.border = 'unset';
+    }
+  }
+
+  const columns = [
+    {
+      name: 'file_name',
```

Comment:
In near future, these should be made configurable

Reply:
I am wait for the user story to be completed, so I know which parts are configurable.  will do those together.

Reply:
You can reuse /src/assets/program/externalLinkIcon.svg icon right?

Reply:
While it is under program folder, I do not think is a good idea that link the cart icon to the program page's icon.  The folder suggests the icon is premiered for program, not the cart.  How about move the icon out of program folder to icons folder and named externalLInkIcon.svg for general use, in both program page, cart page and etc.

Reply:
Thats sounds like a plan!!!